modules/apps/static/portal-configuration/portal-configuration-persistence-impl/src/main/java/com/liferay/portal/configuration/persistence/internal/upgrade/ConfigurationUpgradeStepFactoryImpl.java

```diff
@@ -106,6 +110,10 @@ public UpgradeStep createUpgradeStep(String oldPid, String newPid) {
 
 						_renameConfigurationFile(oldConfigFile, newConfigFile);
 					}
+
+					if (isFactoryPid) {
```

Comment:
@dantewang @yuhai 
Why we skip renaming config file for factory config file?

Tina

Reply:
@tinatian We don't skip renaming config file for factory.pid. It's renamed within the while loop. 

The logic is, if the pid is factory.pid, don't try to rename config files with exactly pid as file name, because for factory.pid the actual config files must have a suffix appended to factory.pid as the file name.